### PR TITLE
Basic config and secret store logic

### PR DESCRIPTION
**Pull request by denisfd**

### CODE REVIEW

File: app/api/v2/resource/base.rb [hunk 1]
```diff
@@ -37,6 +37,11 @@ class Base < Grape::API
         end
       end
```
Comment (mod) [inline app/api/v2/resource/base.rb]: Keep this inside phone API file not shared with other controllers

File: app/api/v2/resource/utils.rb [hunk 2]
```diff
@@ -12,6 +12,20 @@ def current_user
         end
       end
```
Comment (mod) [inline app/api/v2/resource/utils.rb]: I don't think this need to be shared as well.

File: app/helpers/fake_sms.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class FakeSMS
```
Comment (mod) [inline app/helpers/fake_sms.rb]: this would be declined,
is it a test helper ? it should go in spec/helpers

File: app/models/phone.rb [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  before_validation :parse_country
+  before_validation :sanitize_number
+  after_initialize :generate_code
+
+  scope :verified, -> { where.not(validated_at: nil) }
+  scope :kept, -> { joins(:user).where(users: { discarded_at: nil }) }
+
+  def number_exists?
+    Phone.verified.exists?(number: number)
+  end
```
Comment (mod) [inline app/models/phone.rb]: maybe move code related to phone in here:
def phone_valid?(phone_number)

File: config/secrets.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+development:
```
Comment (mod) [inline config/secrets.yml]: secrets.yml don't exists in rails 5.2 you need to use credentials.yml

File: config/initializers/fake_twilio.rb [hunk 6]
```diff
@@ -0,0 +1,3 @@
+if Rails.env.development?
```
Comment (mod) [inline config/initializers/fake_twilio.rb]: centralise both in a single file like phone.rb

File: app/helpers/fake_sms.rb (hunk 3 above)
Comment (denisfd) [reply app/helpers/fake_sms.rb]: It is also used in development

**Pull request merged**